core/build.gradle

```diff
@@ -16,9 +17,13 @@ sourceSets {
 dependencies {
   compileOnly libs.bundles.intelliJ
 
+  testFixturesApi project(":core")
+  testFixturesApi libs.junit
+  testFixturesCompileOnly libs.bundles.intelliJ
```

Comment:
Removed intellij dep as api because we already shade IntelliJ in sqldelight so each dialect needs to exclude this api dependency anyway: see https://github.com/cashapp/sqldelight/blob/91b92a4162ff7af3232c172463244b69a7c70e52/dialects/hsql/build.gradle#L23